service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/util/ReferenceUtil.java

```diff
@@ -0,0 +1,39 @@
+package uk.nhs.adaptors.oneoneone.cda.report.util;
+
+import java.util.Arrays;
+import java.util.function.Predicate;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.hl7.fhir.dstu3.model.DomainResource;
+import org.hl7.fhir.dstu3.model.Reference;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class ReferenceUtil {
```

Comment:
This class is a copy paste, we don't need the entire code.

Reply:
Module removed.